react/features/jaas/functions.js

```diff
@@ -1,9 +1,88 @@
 // @flow
 
-import { getVpaasTenant } from '../billing-counter/functions';
+import { jitsiLocalStorage } from '@jitsi/js-utils';
+import uuid from 'uuid';
 
+import { BILLING_ID, VPAAS_TENANT_PREFIX } from './constants';
 import logger from './logger';
 
+/**
+ * Returns the full vpaas tenant if available, given a path.
+ *
+ * @param {string} path - The meeting url path.
+ * @returns {string}
+ */
+export function extractVpaasTenantFromPath(path: string) {
+    const [ , tenant ] = path.split('/');
+
+    if (tenant.startsWith(VPAAS_TENANT_PREFIX)) {
+        return tenant;
+    }
+
+    return '';
+}
+
+/**
+ * Returns the vpaas tenant.
+ *
+ * @param {Object} state - The global state.
+ * @returns {string}
+ */
+export function getVpaasTenant(state: Object) {
+    return extractVpaasTenantFromPath(state['features/base/connection'].locationURL.pathname);
+}
+
+/**
+ * Returns true if the current meeting is a vpaas one.
+ *
+ * @param {Object} state - The state of the app.
+ * @param {boolean} requiredJwt - Whether jwt is required or not.
+ * @returns {boolean}
+ */
+export function isVpaasMeeting(state: Object, requiredJwt: boolean = true) {
+    const { billingCounterUrl, iAmRecorder, iAmSipGateway } = state['features/base/config'];
+    const { jwt } = state['features/base/jwt'];
+
+    const jwtBoolean = requiredJwt ? Boolean(jwt) : true;
+
+    const isAllowed = iAmRecorder || iAmSipGateway || jwtBoolean;
+
+    return Boolean(
+        billingCounterUrl
+        && extractVpaasTenantFromPath(
+            state['features/base/connection'].locationURL.pathname)
+        && isAllowed
+    );
+}
+
+/**
+ * Returns the stored billing id (or generates a new one if none is present).
+ *
+ * @returns {string}
+ */
+export function getBillingId() {
+    let billingId = jitsiLocalStorage.getItem(BILLING_ID);
+
+    if (!billingId) {
+        billingId = uuid.v4();
+        jitsiLocalStorage.setItem(BILLING_ID, billingId);
+    }
+
+    return billingId;
+}
+
+/**
+ * Returns the billing id for vpaas meetings.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string | undefined}
+ */
+export function getVpaasBillingId(state: Object) {
```

Comment:
are these two used anymore?

Reply:
They're used in the conference options.
![image](https://user-images.githubusercontent.com/11474153/125612513-84e6840c-bc5f-4dcf-9d2a-67892a4127c3.png)

Reply:
By the looks of [this commit](https://github.com/jitsi/jitsi-meet/commit/bfc4b2ac6f0ce0431ede825e38fec1673ab00280) it's a leftover of the previous implementation. @vp8x8  please confirm whether this is the case and if so, whether it is safe to remove the code

Reply:
Yes, this is correct. It can be removed now.

Reply:
Removed it